### ISSUE TITLE
Use CRUX data for token stats page

### ISSUE THREAD

**Issue by esot321c** [paideiadao/paideia-app#148]
/assets/token_stats/{token_id}

example call for walrus dao
https://api.paideia.im/assets/token_stats/59ee24951ce668f0ed32bdb2e2e5731b6c36128748a3b23c28407c5f8ccbf0f6

**Comment by esot321c** [comment]
Use Crux instead

**Comment by noob77777** [comment]
Depends on #171

**Comment by noob77777** [comment]
Closing in favour of #171
